docs/add-merge-automation-to-your-repository.md

```diff
@@ -0,0 +1,38 @@
+# Adding merge automation to your repository
+
+In order to run tests and enable automated merging of pull requests for your repository on KubeVirt [prow] you need to have the following components configured:
+- [tide] - responsible for maintaining the merge pool
+- [lgtm] - sets `lgtm` label
+- approve - sets label `approved`
```

Comment:
https://github.com/kubernetes/test-infra/tree/master/prow/plugins/approve maybe?

Reply:
Yeah, I just skipped that as there's no helpful README.md there. Maybe this one is more helpful, although I don't really like deep links to the code for a document: https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approve.go#L132

Also the lgtm README points towards a much more comprehensive documentation of the workflow.

> Also, GitHub approval auto-adds `lgtm`, might be worth mentioning.

Not in every case, but for our config you are right: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml#L463

Thanks, will add that!